python/cugraph/cugraph/tests/dask/test_mg_bfs.py

```diff
@@ -85,6 +85,70 @@ def modify_dataset(df):
     assert err == 0
 
 
+@pytest.mark.skipif(
+    is_single_gpu(), reason="skipping MG testing on Single GPU system"
+)
+def test_dask_bfs_undirected(dask_client):
+    gc.collect()
+
+    input_data_path = (RAPIDS_DATASET_ROOT_DIR_PATH /
+                       "netscience.csv").as_posix()
+
+    print(f"dataset={input_data_path}")
+    chunksize = dcg.get_chunksize(input_data_path)
+
+    ddf = dask_cudf.read_csv(
+        input_data_path,
+        chunksize=chunksize,
+        delimiter=" ",
+        names=["src", "dst", "value"],
+        dtype=["int32", "int32", "float32"],
+    )
+
+    def modify_dataset(df):
+        temp_df = cudf.DataFrame()
+        temp_df['src'] = df['src']+1000
+        temp_df['dst'] = df['dst']+1000
+        temp_df['value'] = df['value']
+        return cudf.concat([df, temp_df])
+
+    meta = ddf._meta
+    ddf = ddf.map_partitions(modify_dataset, meta=meta)
+
+    df = cudf.read_csv(
+        input_data_path,
+        delimiter=" ",
+        names=["src", "dst", "value"],
+        dtype=["int32", "int32", "float32"],
+    )
+
+    df = modify_dataset(df)
+
+    g = cugraph.Graph(directed=False)
```

Comment:
It looks like the only difference between this test and `test_dask_bfs` is the type of Graph. Can't you just have `directed` as a parameter?

Reply:
Done